docs/en/setup/backend/backend-zabbix.md

```diff
@@ -0,0 +1,67 @@
+# Zabbix Receiver
+Zabbix receiver is accepting the metrics of [Zabbix Agent Active Checks protocol](https://www.zabbix.com/documentation/current/manual/appendix/items/activepassive#active_checks) fromat into the [Meter System](./../../concepts-and-designs/meter.md).
+
+## Module define
+```yaml
+receiver-zabbix:
+  selector: ${SW_RECEIVER_METER:default}
+  default:
+    # Export tcp port, Zabbix agent could connected and transport data
+    port: 13800
+    # Enable config when receive agent request
+    activeFiles: agent.yaml
+```
+
+## Configuration file
+Zabbix receiver is configured via a configuration file. The configuration file defines everything related to receiving 
+ from agents, as well as which rule files to load.
+ 
+OAP can load the configuration at bootstrap. If the new configuration is not well-formed, OAP fails to start up. The files
+are located at `$CLASSPATH/zabbix-receive-config`.
+
+The file is written in YAML format, defined by the scheme described below. Brackets indicate that a parameter is optional.
+
+A example can be found [here](../../../../oap-server/server-bootstrap/src/main/resources/meter-analyzer-config/spring-sleuth.yaml).
+You could find the Zabbix agent detail items from [Zabbix Agent docucment](https://www.zabbix.com/documentation/current/manual/config/items/itemtypes/zabbix_agent).
+
+### Zabbix agent configure
```

Comment:
This title could mislead people thought, this is Zabbix's native configuration. This should be purely `Configuration file`